clarity-ext-scripts/clarity_ext_scripts/covid/rtpcr_analysis_service.py

```diff
@@ -26,82 +33,121 @@ class NegativeControlWasPositive(AnalysisServiceException):
     pass
 
 
+class FailedControl(AnalysisServiceException):
+    pass
+
+
 class RTPCRAnalysisService(object):
+    """
+    Create a new RTPCRAnalysis Service:
+        RTPCRAnalysisService(covid_reporter_key="FAM",
+                             internal_control_reporter_key="HEX")
+    """
 
-    HEX_THRESHOLD = 32
-    FAM_THRESHOLD = 38
+    INTERNAL_CONTROL_THRESHOLD = 32
+    COVID_CONTROL_THRESHOLD = 38
 
-    def __init__(self):
-        pass
+    def __init__(self, covid_reporter_key, internal_control_reporter_key):
+        self._covid_reporter_key = covid_reporter_key
+        self._internal_control_reporter = internal_control_reporter_key
 
-# Understanding of the analysis criterias
-# FAM = COVID, HEX = Human internal control
-# Positive if CT(FAM) = <=38
-# Failed  if CT(FAM)= >38
-# Negative if CT(FAM) = 0 and CT(HEX) = <=32
-# Retest if CT(FAM) = 0 and CT(HEX) = >32
+    # These are notes on what Maike said about the analysis
+    # Understanding of the analysis criterias
+    # FAM = COVID, HEX = Human internal control
+    # Positive if CT(FAM) = <=38
```

Comment:
What does `= <=` mean?

Reply:
I took it straight from Maikes notes, and did not want to edit them (to not risk losing information). But from talking to here I means the same as "positive if CT(FAM) <= 38". The same goes for the other cases.